### PR TITLE
Add Maji CLI convenience commands

### LINKED ISSUE #24: Alias more Maji commands
Currently it's not clear when to use a "Maji" command and when something else. During development these are the most important ones:

```
make watch
bin/karma start
bundle exec rspec
bin/ci
```

Perhaps it's better to alias them all as maji commands, for example:

```
maji watch
maji karma start
maji rspec
maji ci
```

### PR DESCRIPTION
**Pull request by pascalw**
Fixes #24.

@eirvandelden @matthijsgroen please review.

### CODE REVIEW

File: docs/upgrade_guide.md [hunk 1]
```diff
@@ -154,5 +154,31 @@ With Maji 2.x new projects use webpack for module bundling. While not required i
 
 With these steps your app should become functional again, depending on how much custom stuff your app uses (like overwriting private implementations of Marionette).
 
+## Maji CLI
+
+Maji 2.x adds several new subcommands to the Maji CLI, streamlining the development workflow to have a single way to interact with your project:
+
+* `maji start` - starts the development server
+* `maji build` - build static assets
+* ` maji test` - runs your projects tests once
```
Comment (matthijsgroen) [inline docs/upgrade_guide.md]: Space

File: docs/upgrade_guide.md [hunk 2]
```diff
@@ -154,5 +154,31 @@ With Maji 2.x new projects use webpack for module bundling. While not required i
...
+* `maji test --unit` - runs your unit test once
+* `maji test --integration` - runs your integration test once
+* `maji test --watch` - runs your tests whenever files are changed
+
+All these commands delegate to NPM run scripts. When upgrading from Maji 1.x to Maji 2.x you can add the following snippet to your `package.json` to add support for these commands:
+
+```json
+"scripts": {
+  "test": "bin/ci",
+  "test:unit": "APP_ENV=test karma start --single-run",
```
Comment (matthijsgroen) [inline docs/upgrade_guide.md]: Here it is `test:unit`, it is shown as `test:karma` in the template

File: project_template/package.json [hunk 3]
```diff
@@ -2,6 +2,13 @@
   "name": "example",
   "version": "1.0.0",
   "private": true,
+  "scripts": {
+    "test": "bin/ci",
+    "test:karma": "karma start --single-run",
```
Comment (matthijsgroen) [inline project_template/package.json]: Here it is `test:karma`, it is shown as `test:unit` in the documentation

**Comment by eirvandelden** [comment]
@pascalw What @matthijsgroen said. After that, :shipit:

File: docs/upgrade_guide.md (hunk 2 above)
Comment (pascalw) [inline docs/upgrade_guide.md]: Fixed

**Pull request merged**